control-plane/pkg/reconciler/trigger/v2/triggerv2.go

```diff
@@ -60,25 +63,28 @@ const (
 )
 
 type Reconciler struct {
-	BrokerLister        eventinglisters.BrokerLister
-	ConfigMapLister     corelisters.ConfigMapLister
-	EventingClient      eventingclientset.Interface
-	Env                 *config.Env
-	ConsumerGroupLister internalslst.ConsumerGroupLister
-	InternalsClient     internalsclient.Interface
-	SecretLister        corelisters.SecretLister
-	KubeClient          kubernetes.Interface
-	KafkaFeatureFlags   *apisconfig.KafkaFeatureFlags
+	BrokerLister         eventinglisters.BrokerLister
+	ConfigMapLister      corelisters.ConfigMapLister
+	ServiceAccountLister corelisters.ServiceAccountLister
+	EventingClient       eventingclientset.Interface
+	Env                  *config.Env
+	ConsumerGroupLister  internalslst.ConsumerGroupLister
+	InternalsClient      internalsclient.Interface
+	SecretLister         corelisters.SecretLister
+	KubeClient           kubernetes.Interface
+	KafkaFeatureFlags    *apisconfig.KafkaFeatureFlags
+	FeatureStore         feature.Store
 }
 
 func (r *Reconciler) ReconcileKind(ctx context.Context, trigger *eventing.Trigger) reconciler.Event {
-	trigger.Status.MarkOIDCIdentityCreatedNotSupported()
```

Comment:
where do we set this condition now?

Reply:
in `SetupOIDCServiceAccount` ?

Reply:
Yeah, `SetupOIDCServiceAccount` should do that, as far as I know